src/service_identity/_common.py

```diff
@@ -421,3 +423,6 @@ def _validate_pattern(cert_pattern):
 # Ensure no locale magic interferes.
 _TRANS_TO_LOWER = maketrans(b"ABCDEFGHIJKLMNOPQRSTUVWXYZ",
                             b"abcdefghijklmnopqrstuvwxyz")
+
+
+ID_ON_DNS_SRV = ObjectIdentifier('1.3.6.1.5.5.7.8.7')  # id_on_dnsSRV
```

Comment:
Oh, right, I forgot I was the one that moved this, in this branch. (I couldn't use it in the end, because cryptography's `ObjectIdentifier` is not `pyasn1`'s `ObjectIdentifier`). I'll undo that.